DGN/Models/Category.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DGN.Models
+{
+    public class Category
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [DisplayName("Category name")]
+        [Required]
+        public string CategoryName { get; set; }
+
+        [DisplayName("The articles that using this category")]
```

Comment:
Should be: "Articles related to this category"

Reply:
ok bro